swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -133,6 +133,8 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
     adapter?.stop()
 
     cancelTunnelWithError(nil)
+    super.stopTunnel(with: reason, completionHandler: completionHandler)
```

Comment:
Now that we don't erroneously kill the process by needlessly reactivating it, I thought it would be a good idea to add these missing lifecycle callbacks we should have had.

This will provide more context to the system over whether it can reap our process.